superset/db_engine_specs/base.py

```diff
@@ -443,6 +452,9 @@ def get_dbapi_mapped_exception(cls, exception: Exception) -> Exception:
         """
         new_exception = cls.get_dbapi_exception_mapping().get(type(exception))
         if not new_exception:
+            # We are interested in just BigQuery exceptions
+            if cls.engine == "bigquery":
```

Comment:
can we add `bigquery` to this engine + add it here as `Engines.BigQuery`

https://github.com/apache/superset/blob/66c0801e494f16ba44d492f3648c1e1a57c8e35a/superset-frontend/src/views/CRUD/data/database/types.ts#L150

Reply:
Hmm, that's in the `superset-frontend` module, right? Do we have some similar enum in Python?